CONTRIBUTIONS.md

```diff
@@ -0,0 +1,11 @@
+If you'd like your contributions to be displayed in the repo readme, do the following:
+
+In an issue or pull request, simply comment the following: 
+    @all-contributors please add @<github username> for <contribution>
```

Comment:
If this is likely to be viewed as an md, then it looks like anything inside <> gets treated as a comment and removed

Reply:
Adding on this, I suggest using `code blocks`. For example;
`@all-contributors please add @<github username> for <contribution>`

Reply:
Thanks for spotting those, looks like I forgot to check it! I've made those changes now.